administrator/com_joomgallery/src/Service/Uploader/Uploader.php

```diff
@@ -244,8 +244,11 @@ public function overrideData(&$data): bool
     // Create the IMGtools service
```

Comment:
When I see this correct, then the IMGtools service is not needed anymore in this method Uploader::overrideData().
Therefore createIMGtools() and delIMGtools() could be deleted in this method.